tests/system/CommonFunctionsTest.php

```diff
@@ -541,12 +541,34 @@ public function testDWithCSP()
      */
     public function testTraceWithCSP()
     {
+        $this->resetServices();
+
         /** @var App $config */
-        $config              = config(App::class);
+        $config              = config('App');
         $config->CSPEnabled  = true;
         Kint::$cli_detection = false;
 
-        $this->expectOutputRegex('/<style {csp-style-nonce} class="kint-rich-style">/u');
+        $this->expectOutputRegex('/<style nonce="[0-9a-z]{24}" class="kint-rich-style">/u');
         trace();
     }
+
+    public function testCspStyleNonce()
+    {
+        $this->resetServices();
+
+        $config             = config('App');
+        $config->CSPEnabled = true;
+
+        $this->assertStringContainsString('nonce="', csp_style_nonce());
```

Comment:
Can you change to `assertStringStartsWith`?

Reply:
Thanks! Fixed.